### ISSUE TITLE
scrollToBottom not working on linux

### ISSUE THREAD

**Issue by Wh1teDuke** [trustable-code/NiGui#120]
gtk debug output prints `_gtk_text_view_scroll_to_iter: assertion 'xalign >= 0.0 && xalign <= 1.0' failed`

**Comment by trustable-code** [comment]
For me it works, can you provide more details please?

**Comment by Wh1teDuke** [comment]
> For me (Manjaro with Xfce) it works, can you provide more details please?
lxde libgtk-3-0:i386  3.24.24-1

```nim
import nigui

app.init()
var window = newWindow("NiGui Example")
var container = newLayoutContainer(Layout_Vertical)
window.add(container)

var textArea = newTextArea()
container.add(textArea)

discard startRepeatingTimer(300, proc(event: TimerEvent) =
  textArea.addLine($len(textArea.text))
  scrollToBottom(textArea)
)

window.show()
app.run()
```

This snippet doesn't work as expected for me.

![nigui](https://user-images.githubusercontent.com/5203140/103876987-ba129180-50d4-11eb-833c-ce50c795ce9c.gif)

**Comment by trustable-code** [comment]
I guess it is fixed with https://github.com/trustable-code/NiGui/commit/5db6f02cadaa522caf2814c360669ba94fb9c57a .
The wrong data type affected only 32-bit systems and not 64-bit systems.

**Comment by Wh1teDuke** [comment]
> I guess it is fixed by [5db6f02](https://github.com/trustable-code/NiGui/commit/5db6f02cadaa522caf2814c360669ba94fb9c57a) .
> The wrong data type affected only 32-bit systems and not 64-bit systems.

Yes, that was it. Thank you for fixing my issue.
